templates/shared/issuelist.tmpl

```diff
@@ -59,6 +59,29 @@
 					{{else}}
 						{{$.i18n.Tr .GetLastEventLabelFake $timeStr (.Poster.GetDisplayName | Escape) | Safe}}
 					{{end}}
+					{{if and .IsPull}}
+						<div class="branches">
+							<div class="branch">
+								<a href="/{{if .PullRequest.BaseRepo}}{{.PullRequest.BaseRepo.OwnerName}}/{{.PullRequest.BaseRepo.LowerName}}{{end}}/src/branch/{{.PullRequest.BaseBranch}}">
+									{{if ne .RepoID .PullRequest.BaseRepoID}}
+										<div class="truncated-name dib vb">{{.PullRequest.BaseRepo.OwnerName}}</div><div class="truncated-name dib vb">:</div>
+									{{end}}
+									<div class="truncated-name dib vb">{{.PullRequest.BaseBranch}}</div>
+								</a>
+							</div>
+							<div class="pointer">
+								{{svg "gitea-double-chevron-left" 13  }}
```

Comment:
All calls to `svg` are using 14 or 16 in existing code. Is the 13 chosen by purpose?

Reply:
Set to 14